src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -134,15 +134,16 @@ class PolicyHandlerTest : public ::testing::Test {
  protected:
   NiceMock<MockPolicySettings> policy_settings_;
   NiceMock<event_engine_test::MockEventDispatcher> mock_event_dispatcher_;
-  std::shared_ptr<application_manager_test::MockApplication> mock_app_;
-  connection_handler_test::MockConnectionHandler conn_handler;
-  protocol_handler_test::MockSessionObserver mock_session_observer;
-  application_manager_test::MockStateController mock_state_controller;
+  std::shared_ptr<NiceMock<application_manager_test::MockApplication> >
+      mock_app_;
+  NiceMock<connection_handler_test::MockConnectionHandler> conn_handler;
+  NiceMock<protocol_handler_test::MockSessionObserver> mock_session_observer;
+  NiceMock<application_manager_test::MockStateController> mock_state_controller;
   components::usage_statistics_test::MockStatisticsManager
       mock_statistics_manager_;
   PolicyHandler policy_handler_;
   std::shared_ptr<policy_manager_test::MockPolicyManager> mock_policy_manager_;
-  application_manager_test::MockApplicationManager app_manager_;
+  NiceMock<application_manager_test::MockApplicationManager> app_manager_;
```

Comment:
@ValeriiMalkov A lot of using `application_manager_test`. Can we make `using namespace` above?